docs/sources/articles/networking.md

```diff
@@ -186,7 +186,20 @@ Four different options affect container domain name services.
 Note that Docker, in the absence of either of the last two options
 above, will make `/etc/resolv.conf` inside of each container look like
 the `/etc/resolv.conf` of the host machine where the `docker` daemon is
-running.  You might wonder what happens when the host machine's
+running.  The daemon will filter out any localhost nameservers in
```

Comment:
Thanks for the add Phil. The note may make more sense  incorporated into the text about filtering.  The real orthogonal info is the bit about the resolver into a note.  The text switches between Docker and "docker daemon" -- if it is technically correct to say the daemon adds google...then do that.

Below is my suggestion for cleaning up this small bit... 

-----------------------

Docker, in the absence of either `--dns-search=DOMAIN...` or `--dns=IP_ADDRESS...`, makes each container's `/etc/resolv.conf` look like
the `/etc/resolv.conf` of the host machine (where `docker` daemon runs). When creating the container's copy, the daemon filters out the localhost
nameservers from the host's `/etc/resolv.conf`. 

Filtering is necessary because the localhost is unreachable from the container's
network.  After this filtering if there are no more `nameserver` entries left in
the `/etc/resolv.conf` file, the daemon adds public Google DNS nameservers
(8.8.8.8 and 8.8.4.4) into the container's DNS configuration.

> **Note:** If you need access to a host's localhost resolver, you must modify
> your DNS service on the host to listen on a non-localhost address that is
> reachable from within the container.